### PR TITLE
<3

### PR DESCRIPTION
**Pull request by fedeGalvan**

### CODE REVIEW

File: src/components/TorrentCard/TorrentCard.js [hunk 1]
```diff
@@ -70,8 +70,8 @@ const TorrentCard = ({ torrent }) => {
         </MainInfoContent>
         <ProgressBarContainer>
           <CircularProgressbar
-            value={calcPercentage(torrent.progress)}
-            text={calcPercentage(torrent.progress)}
+            value={calcPercentage(torrent.progress)  + "%"}
+            text={calcPercentage(torrent.progress) + "%"}
```
Comment (nuzzesick) [inline src/components/TorrentCard/TorrentCard.js:74]: would be nice if you replace this with: ${calcPercentage(torrent.progress)}%

File: src/pages/Home/Home.js [hunk 2]
```diff
@@ -1,34 +1,21 @@
...
-      <HomePageContainer onClick={() => {
-        if (torrentSelected) {
-          clearTorrentSelection();
-        }
-      }}
-      >
-        <TorrentsList
-          activeFilter={activeFilter}
-        />
+      <HomePageContainer onClick={() => clearTorrentSelection()}>
```
Comment (nuzzesick) [inline src/pages/Home/Home.js:15]: would be cool if you can replace this with `<HomePageContainer onClick={clearTorrentSelection}>`

File: src/helpers/when.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+'use strict'
+
+const isFunction = (input) => typeof input === 'function';
+
+export default (predicate) => (elemOrThunk) =>
```
Comment (nuzzesick) [inline src/helpers/when.js:5]: im not sure about this 'when' function, why to use it? how can improve our performance/code using it?

Comment (fedeGalvan) [reply src/helpers/when.js:5]: Just to keep the logic of the component above the return and the view inside. No performance improvment

**Pull request merged**